vyked/registry.py

```diff
@@ -216,7 +216,17 @@ def receive(self, packet: dict, protocol, transport):
         elif request_type == 'pong':
             self._ping(packet)
         elif request_type == 'ping':
-            self._pong(packet, protocol)
+            if 'payload' in packet:
```

Comment:
Yes, but you are checking if a node_id is valid or not before responding which is what the flag 'to_send' is for. A better name might be is_valid_node for the flag.